testlib/src/test/resources/license/LicenseHeaderWithYearToken

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) $YEAR. ACME corp.
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ **/
```

Comment:
Just the concept of having a GNU licence anywhere in the source code is kinda a red flag.

Since the GNU license itself is licensed under GNU and we are "linking" (can be interpreted broadly) against the licence by reading in the file perhaps it would be better to pick a different licence with a year token.

Reply:
Great catch!

Reply:
While I totally respect GNU I don't want it anywhere near my software. 😆

Reply:
haha, I will change that!